x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/search_index.tsx

```diff
@@ -151,8 +151,9 @@ export const SearchIndex: React.FC = () => {
       pageHeader={{
         pageTitle: indexName,
         rightSideItems: [
-          ...headerActions,
           ...(isCrawlerIndex(indexData) ? [<CrawlerStatusIndicator />] : []),
+          // ...(isConnectorIndex(indexData) ? [<ConnectorSyncButton />] : []),
```

Comment:
We already have a sync button, it's just in the header actions component rather than in here. So this comment should probably be removed.

Reply:
Or we should move that component here 🤔

Reply:
Whoops, I wasn't sure if sync had been merged so I left this for future conflict resolution